ClientLib/src/main/java/com/o3dr/services/android/lib/coordinate/LatLongAlt.java

```diff
@@ -14,24 +15,34 @@
      * Stores the altitude in meters.
      */
     private double mAltitude;
+    private Frame mFrame;
+
+    public LatLongAlt() {
+        super();
+        mAltitude = 0.0;
+        mFrame = Frame.GLOBAL_RELATIVE;
+    }
 
-    public LatLongAlt(double latitude, double longitude, double altitude) {
+    public LatLongAlt(double latitude, double longitude, double altitude, Frame frame) {
```

Comment:
@billbonney Given that `Frame.GLOBAL_RELATIVE` is the default frame, you could include an additional constructor that omit the frame parameter and sets `mFrame` to `Frame.GLOBAL_RELATIVE`. 
This removes the need to update all the files where the constructor is invoked.